### ISSUE TITLE
[UI] All text component disappear when using a TextEntry placeholder

### ISSUE THREAD

**Issue by bub-bl** [Facepunch/sbox-issues#6774]
### Describe the bug

When we set a Placeholder on the TextEntry component, all text components disappear randomly for no reason

### To Reproduce

- Create some text components like span, label, etc..
- Add a TextEntry and set its placeholder
- Start the scene and play with some styling, you should be seeing text components disappear after few modifications

### Expected behavior

Text components should be visible when setting the Placeholder on a TextEntry

**Comment by CarsonKompon** [comment]
Could you include a video/repro? Would be really helpful when debugging this

**Comment by bub-bl** [comment]
> Could you include a video/repro? Would be really helpful when debugging this

This problem is quite random, sometimes it happens every time, sometimes it doesn't... I don't know what is causing this problem. It's definitely caused by the TextEntry component, but the problem appears randomly after updating some css styles related to TextEntry itself.

You can skip over 1:40 if you dont want to wait until the bug appearing :D

In the video, I make numerous changes to the TextEntry until the text disappears.
Delete the TextEntry, saving and re-adding it + save solve the problem

The video is not really helpfull, but i cant find a repro that work everytime the first time
https://streamable.com/xaovzn

**Comment by bub-bl** [comment]
Today, i have started my project and i get no text again.. I have removed the TextEntry, restarted the editor and the problem still here and i don't know why..
I don't have any errors, any logs. All components work except text component like span, label, etc

Now, im not sure the problem are caused by the TextEntry

I have deleted sbox from steam, removed sbox folder and reinstalled it. Seem to work now, but why ???
I can't help you more, i don't understand the problem and its not caused by my code, i have the same problem randomly on new project

**Comment by MrBrax** [comment]
Are you using css `transform` at all?

**Comment by bub-bl** [comment]
> s `transform` at all?

Yeah, i use a lot of transform for transition. I have tried to removed all the transform in my code and restarted sbox, but i lose my text components, again...

**Comment by bub-bl** [comment]
Now my text appears again, wtf....
I'm going to spend a few hours without transformation to see if it's due to this.

I've had this kind of problem before with scaling transformations that made components invisible when downscaling

**Comment by MrBrax** [comment]
transform has always had issues for me for some reason or another, can't rely on it at all for centering

**Comment by bub-bl** [comment]
After 1 hours, i get the problem again after removing TextEntry and transforms

**Comment by bub-bl** [comment]
@CarsonKompon I've found the problem. It occurs when a component is called with the same name as an existing component in the Sandbox.UI library.
In my case, I created a component called `Label` in the Project.Components namespace. It randomly came into conflict with the `Sandbox.UI.Label` component during HotReload. As the label is a text, all text components stopped working.

If you still want to solve the problem, you know where to look :)
